xanalytics/management/commands/populate_xanalytics.py

```diff
@@ -0,0 +1,42 @@
+"""Management command to populate xanalytics data by course ID."""
+
+import logging
+from random import randint
```

Comment:
Would it be possible to seed random first, for example  `Random(0).randint(2,3)`? That way we have reproduceable data